docs/source/data_collator.rst

```diff
@@ -0,0 +1,48 @@
+<!---
+     Copyright 2020 The HuggingFace Team. All rights reserved.
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+-->
+
+DataCollator
+-----------------------------------------------------------------------------------------------------------------------
+
+DataCollators are objects that will form a batch by using a list of elements as input. These lists of elements are of
+type :obj:`train_dataset` or :obj:`eval_dataset`.
+
+The base class :class:`~transformers.data.data_collator` can use the default
```

Comment:
This class does not exist. Did you mean the `data_collator` argument in Trainer?